java/query/builder/Aggregatable.java

```diff
@@ -31,50 +31,26 @@ default T count() {
         return aggregate(TypeQLToken.Aggregate.Method.COUNT, null);
     }
 
-    default T max(String var) {
```

Comment:
Disable string interface since "x" could now refer to "$x" or "?x"

Reply:
In a lot of places I just default it to the concept variable. I can go about fixing that